storage/tianmu/common/common_definitions.cpp

```diff
@@ -40,43 +40,51 @@ void PushWarningIfOutOfRange(THD *thd, std::string col_name, int64_t v, int type
   // below `0` is for min unsigned value.
   switch (type) {
     case 1: {  // MYSQL_TYPE_TINY
```

Comment:
Would you like change these magic number by the way.

Reply:
Yes, I'll fix it.